### PR TITLE
[#426] fixed undefined role message (connects #426)

### PR DESCRIPTION
**Pull request by MacNew**

### CODE REVIEW

File: src/app/courses/courses.component.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ export class CoursesComponent implements OnInit, AfterViewInit {
         }).filter((c: any) => {
           return c._id !== '_design/course-validators';
         });
-      });
+      }, (error) => this.planetMessageService.showAlert('There was a problem getting courses please, contact admin'));
```
Comment (lmmrssa) [inline src/app/courses/courses.component.ts]: make it 'There was a problem getting courses.'

File: src/app/meetups/meetups.component.ts [hunk 2]
```diff
@@ -39,7 +39,7 @@ export class MeetupsComponent implements OnInit, AfterViewInit {
         // _all_docs returns object with rows array of objects with 'doc' property that has an object with the data.
         // Map over data.rows to remove the 'doc' property layer
         this.meetups.data = data.rows.map(meetup => meetup.doc);
-      }, (error) => this.message = 'There was a problem getting meetups');
+      }, (error) => this.planetMessageService.showAlert('There was a problem getting meetups please, contact admin'));
```
Comment (lmmrssa) [inline src/app/meetups/meetups.component.ts]: remove please contact admin

File: src/app/users/users.component.ts [hunk 3]
```diff
@@ -56,7 +58,7 @@ export class UsersComponent implements OnInit, AfterViewInit {
       this.initializeData();
     } else {
       // A non-admin user cannot receive all user docs
-      this.message = 'Access restricted to admins';
+      this.planetMessageService.showAlert('There was a problem getting members please, contact admin');
```
Comment (lmmrssa) [inline src/app/users/users.component.ts]: Change message to old one 'Access restricted to  Admins only'

File: src/app/courses/courses.component.ts (hunk 1 above)
Comment (MacNew) [reply src/app/courses/courses.component.ts]: Ok

**Pull request merged**